help/help.txt

```diff
@@ -57,6 +58,9 @@
 
     - Prettify logs but don't print pid and hostname
     $ cat log | pino-pretty -i pid,hostname
+    
+    - Prettify logs but only print time and level
+    $ cat log | pino-pretty -i time,level
```

Comment:
i'm sure the example should be here `-I` not `-i`

Reply:
Thanks for catching that! Would you wanna send a PR for that? If not, i can fix that.

Reply:
i don't wanna send pr :)

Reply:
PR up in #375